### DISCUSSION TITLE
How to import island plugins?

### DISCUSSION THREAD

**Discussion by CarcajadaArtificial** [freshframework/fresh#2408] [Q&A]
I created a plugin that adds an island, but I'm unable to import it. According to the [docs](https://fresh.deno.dev/docs/concepts/plugins#islands): 
> will treat them the same way as if they had been placed inside the islands/ directory

But when I try to import the island from a route it using `import Chiplist from '../../islands/Chiplist.tsx'` I get a `Module not found` error. When running the build task, everything seems to work fine regarding the creation and execution of the plugin.

This is how I'm adding the island to my plugin.
```ts
export default function lunchboxPlugin(): Plugin {
  return {
    name: 'lunchbox',
    islands: {
      baseLocation: import.meta.url,
      paths: [
        './src/components/organisms/Chiplist/Chiplist.tsx',
      ],
    },
  };
}
```

Thank you

**Comment by CAYdenberg** [comment]
If you import ChipList from it's actual location (rather than the islands directory), does Fresh treat it _as an island?_ As in, does it get hydrated, behave interactively, etc..?
